enchanted-swiftie/cogs/songs.py

```diff
@@ -0,0 +1,66 @@
+import billboard
```

Comment:
read https://www.python.org/dev/peps/pep-0008/#imports
then press Alt+Ctrl+O in pycharm in this file